volatility3/framework/plugins/windows/mftscan.py

```diff
@@ -98,23 +120,24 @@ def enumerate_mft_records(
         )
 
         # get each of the individual Field Sets
-        mft_object = symbol_table + constants.BANG + "MFT_ENTRY"
-        attribute_object = symbol_table + constants.BANG + "ATTRIBUTE"
+        mft_object_typ_name = symbol_table + constants.BANG + "MFT_ENTRY"
+        attribute_object_typ_name = symbol_table + constants.BANG + "ATTRIBUTE"
 
-        record_map = {}
+        record_map: DefaultDict[str, MFTRecord] = DefaultDict(MFTRecord)
```

Comment:
I'm a little concerned about the use of `DefaultDict` here?  When are we creating blank `MFTRecords`, and does that not mean we need more code to cope with them if we ever look them up?